source/core/assets/images/mpc/finalAtlas/gold_6_buff_to_be_tested/mpcAnimation_6.atlas

```diff
@@ -0,0 +1,1196 @@
+
+mpcAnimation_6.png
+size: 8192, 8192
+format: RGBA8888
+filter: Nearest, Nearest
+repeat: none
+main_player_attack
+  rotate: false
+  xy: 3260, 5433
+  size: 541, 541
+  orig: 541, 541
+  offset: 0, 0
+  index: 1
+main_player_attack
+  rotate: false
+  xy: 3803, 4347
+  size: 541, 541
+  orig: 541, 541
+  offset: 0, 0
+  index: 3
+main_player_attack
+  rotate: false
+  xy: 3803, 1632
+  size: 541, 541
+  orig: 541, 541
+  offset: 0, 0
+  index: 2
+main_player_attack_dizzy
+  rotate: false
+  xy: 3805, 7605
+  size: 541, 541
+  orig: 541, 541
+  offset: 0, 0
+  index: 1
+main_player_attack_dizzy_2.png
```

Comment:
Error over here? Take a look at the sprite naming convention for main_player_attack_dizzy

Reply:
@prateek45 @RecklessPotcover

Reply:
@kprohith The bug is fixed now.